src/main/kotlin/db/migration/data/dev/R__2_5_Integration_test_data.kt

```diff
@@ -327,20 +364,35 @@ class R__2_5_Integration_test_data : BaseJavaMigration() {
           profileId = it.value,
           fromAgencyId = "FROM_AGENCY",
           toAgencyId = "TO_AGENCY",
-          type = MoveType.CANCELLED
+          type = MoveType.CANCELLED,
+          cancellationReason = Move.CANCELLATION_REASON_CANCELLED_BY_PMU,
+          status = MoveStatus.cancelled
         ),
         template
       ).also { move ->
-        create(moveAcceptEvent(move), template)
-        create(moveCancelEvent(move), template)
-        create(
-          journey(move, fromAgencyId = "FROM_AGENCY", toAgencyId = "TO_AGENCY", state = JourneyState.cancelled),
-          template
-        )
+        move.copy(
+          events = listOf(
+            create(moveAcceptEvent(move), template),
+            create(moveCancelEvent(move), template)
+          ),
+          journeys = listOf(
+            create(
+              journey(move, fromAgencyId = "FROM_AGENCY", toAgencyId = "TO_AGENCY", state = JourneyState.cancelled),
+              template
+            )
+          )
+        ).failMigrationIfNotMoveType(MoveType.CANCELLED)
       }
     }
   }
 
+  private fun Move.failMigrationIfNotMoveType(moveType: MoveType) {
```

Comment:
This checks a move that has been created is in the correct state.  If not then the migration will fail.